### ISSUE TITLE
Attempting to run ff9, "couldn't communicate with helper application"

### ISSUE THREAD

**Issue by GFleishman** [OpenEmu/OpenEmu#2839]
Hi!

I would like to run OpenEmu 1.0.4 as a playstation emulator (and eventually for other systems) on my MacBook running OS 10.10.5.

I have installed the appropriate playstation BIOS files according to: https://www.youtube.com/watch?v=6z2Aa3lrDmM

I dragged a folder containing (only) a .bin/.cue pair for FF9 disk 1. The game seems to "import" correctly and the logo for the game appears. When I first double clicked to launch the game, it told me to download the mednafen core, which I did.

Now, when I attempt to launch the game, I get the "couldn't communicate with helper application" window.

Can we troubleshoot the compatibility problem?

**Comment by GFleishman** [comment]
Update: I thought this might have been an "order of installation problem" (like the core needs to be in place before importing the bios files?). I attempted the following, it _did not_ resolve the problem.

I removed the OpenEmu application folder, and ~/Library/Application Support/OpenEmu completely.

I extracted a new copy of OpenEmu, moved it to Applications folder.

I went to OpenEmu -> Preferences -> Cores and installed the mednafen core

I double checked the md5 hashes for my bios files, they are currently named 'SCPH5500.BIN', 'SCPH5501.BIN', and 'SCPH5502.bin'

I dragged them into the OpenEmu window (with playstation selected of course)

I dragged the FF9 disk 1 folder into the window

I double click the game, and still get the "couldn't communicate... " error message.
